src/devhub/entity/community/Card.jsx

```diff
@@ -4,118 +4,68 @@ if (!href) {
   return <></>;
 }
 
-const CommunityCard = ({
-  format,
-  isBannerEnabled,
-  metadata,
-  ...otherProps
-}) => {
-  const renderFormat =
-    format === "small" || format === "medium" ? format : "small";
+const Card = styled.div`
```

Comment:
I like this refactor/simplification

Reply:
Thanks, it wasn't being used anywhere else so it made no sense to be such complex.